docs/site/angular-auth-oidc-client/docs/documentation/configuration.md

```diff
@@ -150,207 +151,214 @@ Starts the OpenID session management for this client.
+Extra parameters that can be added to the token URL request.
 
 ### `disableRefreshIdTokenAuthTimeValidation`
 
 - Type: boolean
 - Required: false
 
-disables the auth_time validation for id_tokens in a refresh due to Azure incorrect implementation
+Disables the `auth_time` validation for `id_tokens` in a refresh due to Microsoft Azure's incorrect implementation.
 
 ### `enableIdTokenExpiredValidationInRenew`
 
 - Type: boolean
 - Required: false
 
-enables the id_token validation, default value is true. You can disable this validation if you would like to ignore the expired value in the renew process. If no id_token is returned in using refresh tokens, set this to false.
+Enables the `id_token` validation. You can disable this validation if you would like to ignore expired values during the renew process. If no `id_token` is returned while using refresh tokens, set this to false. <br/>
+Default = *true*
 
 ### `eagerLoadAuthWellKnownEndpoints`
 
 - Type: boolean
 - Required: false
 
-Tells if the AuthWellKnownEndpoints should be loaded on start or when the user calls the `authorize` method
+Denotes whether `AuthWellKnownEndpoints` should be loaded on start or when the user calls the `authorize` method.
 
 ### `tokenRefreshInSeconds`
 
 - Type: number
 - Required: false
 
-Controls the periodic check time interval in seconds, default = 3
+Controls the periodic check time interval in seconds. <br/>
+Default = *3*
 
 ### `secureRoutes`
 
 - Type: `string[]`
 - Required: false
 
-Array of secure urls on which the token should be send if the interceptor is added to the HTTP_INTERCEPTORS see [Http Interceptor](./using-access-tokens.md/#http-interceptor)
+An array of secure urls to which the token should be sent if the interceptor is added to the `HTTP_INTERCEPTORS`. <br/>
+See [Http Interceptor](./using-access-tokens.md/#http-interceptor)
 
 ### `usePushedAuthorisationRequests`
 
 - Type: boolean
 - Required: false
 
-activates Pushed Authorisation Requests for login and popup login (Iframe renew not supported)
+Activates Pushed Authorisation Requests (PAR) for login and popup login. <br/> 
+(iframe renew is not supported)
 
 ### `refreshTokenRetryInSeconds`
 
 - Type: number
 - Required: false
 
-Controls the periodic retry time interval for retrieving new tokens in seconds, default = 3. `silentRenewTimeoutInSeconds` and `tokenRefreshInSeconds` are upper bounds for this value.
+Controls the periodic retry time interval for retrieving new tokens in seconds. <br/>
+Default = *3*. <br/>
+`silentRenewTimeoutInSeconds` and `tokenRefreshInSeconds` are the upper bounds for this value.
 
 ### `ngswBypass`
 
 - Type: boolean
 - Required: false
 
-Adds the `ngsw-bypass` param to all requests ([Angular Docu](https://angular.io/guide/service-worker-devops#bypassing-the-service-worker)).
+Adds the `ngsw-bypass` param to all requests ([Angular Documentation](https://Angular.io/guide/service-worker-devops#bypassing-the-service-worker)).

```

Comment:
`Angular` --> `angular`. Is not important as URLs are case insensitive but could make an SEO difference.

Reply:
Whoops, same replace-all issue from above.